### ISSUE TITLE
Test module SAM via UART?

### ISSUE THREAD

**Issue by tuananh-lab** [killergeek/nard#3]
My board contain module SAM SEC1210 only support UART.
I created /dev/ttyX via 2 GPIO ports to test communicate with module SAM.
Now what can I do next?

**Comment by killergeek** [comment]
the way of communications is a bit hidden but its in the manual and here
https://microchip.my.site.com/s/article/SEC1210-URT--Minimal-UART-Testing-on-Windows

you need to send first a string of commands to instruct what the sec 1210 needs to do then you send the string with data for the sim/sam module.
depeding on what sim/sam module you have you need that datasheet to see what you need to send to it to work with it.

you can have a look at seader to get a understanding of how to interact with sam's
https://seader.ericbetts.dev/

**Comment by killergeek** [comment]
and if you need a usb connection the header J5 has USB on it.

**Comment by tuananh-lab** [comment]
Thank you!
Do you have any documents on porting CCID, PC/SC or communication, application development via UART port for SAM SEC1210 URT on Android?

**Comment by killergeek** [comment]
no, i have made the NARD to beable to work with sim/sam modules. otheres have made software to work with this hardware.
